apps/gnome-mobile/src/store/useGnomeStore.ts

```diff
@@ -48,4 +57,24 @@ export const useGnomeStore = create<GnomeState>((set) => ({
     set((state) => ({
       gnomes: state.gnomes.filter((gnome) => gnome.id !== id),
     })),
+
+  fetchMyInteractions: async () => {
+    set({ loading: true, error: null });
+    try {
+      const { user } = useAuthStore.getState();
+      const userId = user?.id;
+      if (!userId) {
+        throw new Error("User  is not authenticated");
+      }
+      const data = await (GnomesService as any).getMyGnomesInteractions(userId);
```

Comment:
tu chyba może zostać: 
```suggestion
      const data = await GnomesService.getMyGnomesInteractions(userId);
```